code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -0,0 +1,69 @@
+//night of the unliving deceased
+/mob/living/simple_animal/hostile/zombie
+	name = "zombie"
+	desc = "Don't let it touch you."
+	icon_state = "zombie"
+	icon_living = "zombie"
+	icon_dead = "zombie_dead"
+	speak_emote = list("groans")
+	emote_hear = list("groans")
+	speak_chance = 5
+	turns_per_move = 5
+	see_in_dark = 10
+	response_help  = "pokes the"
+	response_disarm = "shoves aside the"
+	response_harm   = "kicks the"
+	stop_automated_movement_when_pulled = 0
+	maxHealth = 250
+	health = 250
+	melee_damage_lower = 10
+	melee_damage_upper = 40
+	var/poison_per_bite = 5
+	var/poison_type = "blackgoo"
+	faction = FACTION_ZOMBIE
+	move_to_delay = 6
+	speed = 3
+	break_stuff_probability = 75
+
+/mob/living/simple_animal/hostile/zombie/FindTarget()
+	. = ..()
+	if(.)
+		roar_emote()
+
+/mob/living/simple_animal/hostile/zombie/AttackingTarget()
+	..()
+	var/mob/living/target_mob = target_mob_ref?.resolve()
+	if(isliving(target_mob))
+		if(target_mob.reagents)
+			if(prob(25))
+				target_mob.reagents.add_reagent("blackgoo", poison_per_bite)
+				if(prob(poison_per_bite))
+					to_chat(target_mob, SPAN_DANGER("You feel a tiny prick."))
+					target_mob.reagents.add_reagent(poison_type, 5)
+					roar_emote()
+
```

Comment:
looks like you're adding the poison twice here, the first check is 25% and adds poison_per_bite (5) units, then if you pass that check there's a prob(poison_per_bite) so a 5% chance to notify them they were injected and add 5 more units and roar-- not sure I understand why it's adding reagents twice.